src/axom/slic/internal/stacktrace.cpp

```diff
@@ -122,9 +125,13 @@ std::string demangle( char* backtraceString, int frame )
     *returnOffset = 0;
     ++returnOffset;
 
-    int status;
+    int status = false;
+#if !defined(_LIBCPP_VERSION)
     char* realName = abi::__cxa_demangle( mangledName, nullptr, nullptr,
                                           &status );
+#else
+    char* realName = mangledName;
+#endif

```

Comment:
One obvious place to start is:

```diff
// at line 147
-     free(realName);
+ #if !defined(_LIBCPP_VERSION)
+     free(realName);
+ #endif
```

i.e. only free the string if we've allocated it (via ``_cxa_demangle``)